CMake/add_project_kwiver.cmake

```diff
@@ -45,8 +47,10 @@ ExternalProject_Add(kwiver
     -DKWIVER_ENABLE_SERIALIZE_PROTOBUF:BOOL=ON
     -DKWIVER_ENABLE_SERIALIZE_JSON:BOOL=ON
     -DKWIVER_ENABLE_ZeroMQ:BOOL=ON
-    -DKWIVER_INSTALL_SET_UP_SCRIPT:BOOL=ON
+    -DKWIVER_INSTALL_SET_UP_SCRIPT:BOOL=OFF
```

Comment:
This should most likely depend on SKBUILD since we only want to get rid of install script when scikit build is used to create a wheel

Reply:
Gotcha, I'll put a conditional in based on SKBUILD